### PR TITLE
Optimize build size

### PR DESCRIPTION
**Pull request by sarathms**
This attempts to reduce the production build size further from what was done along with the `3.1.0` release. 

~It also upgrades the core dependencies like `electron` to latest versions.~ I had to upgrade electron to `11.x` and then downgrade back to `8.x` because there were breaking changes in core areas like IPC that need to be addressed before we can safely upgrade.

Otherwise this improves Sentry integration, upgrades nextjs to the latest version and gets rid of a lot of unused dependencies.

This also addresses the next.js deprecation notice about `static` dir as described [here](https://github.com/vercel/next.js/blob/master/errors/static-dir-deprecated.md).

Fixes ooni/probe#1151

### CODE REVIEW

**Comment by sarathms** [comment]
Currently, sentry configuration is broken.

File: renderer/components/IntlProvider.js [hunk 1]
```diff
@@ -19,14 +19,14 @@ const IntlProvider = ({ children }) => {
...
-  require(`@formatjs/intl-displaynames/locale-data/${activeLang}`)
+  // require(`@formatjs/intl-displaynames/locale-data/${activeLang}`)
 
   const messages = getMessages(activeLang)
 
   const intl = createIntl({ locale: activeLang, messages }, cache)
 
   const changeLocale = (locale) => {
-    require(`@formatjs/intl-displaynames/locale-data/${locale}`)
+    // require(`@formatjs/intl-displaynames/locale-data/${locale}`)
```
Comment (hellais) [inline renderer/components/IntlProvider.js:29]: These should not be commented out.

**Review by hellais — APPROVED** [review]
LGTM

File: renderer/pages/_app.js [hunk 2]
```diff
@@ -7,7 +7,8 @@ const MyApp = ({ Component, pageProps, err }) => {
 
   // Workaround for https://github.com/zeit/next.js/issues/8592
   const modifiedPageProps = { ...pageProps, err }
-  initSentry()
+
+  // initSentry()
```
Comment (hellais) [inline renderer/pages/_app.js:11]: Why is this commented out?

File: renderer/components/initSentry.js [hunk 3]
```diff
@@ -1,32 +1,48 @@
 import * as Sentry from '@sentry/node'
 import { RewriteFrames } from '@sentry/integrations'
+// import { ipcRenderer } from 'electron'
```
Comment (hellais) [inline renderer/components/initSentry.js:3]: This should be uncommented and reverted the `require` import call.

**Review by hellais — APPROVED** [review]
LGTM

**Pull request merged**